src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,26 @@
+const ArticleCard = props => {
+  const {article_img_url, author, comment_count, created_at, title, topic} = props;
+  return (
+    <div className="article-card">
+      <div className="article-main">
+        <div className="article-header">
+          <div className="names">
+            <h3>{title}</h3>
+            <p className="author-name">By: {author}</p>
+          </div>
+          <div className="details">
+            <p className="date-posted">{created_at}</p>
+            <p className="topic">{topic}</p>
+          </div>
+        </div>
+        <div className="article-footer">
+          <p className="likes-count">0 Likes</p>
+          <p className="comment-count">{comment_count} Comments</p>
+        </div>
+      </div>
+      <img src={article_img_url} alt="article image" />
+    </div>
```

Comment:
too many divs! Please reduce this down to one - have a look into other html tags that are semantic :)

Reply:
I reduced the divs down as much as it made sense to me. the inner divs are purely for styling purposes and i could not find better tags for them.

Reply:
I think the inner divs for styling could be sections instead? Then make your bigger section ( on article list an 'article' instead)? > notes on this: https://www.w3schools.com/tags/tag_article.asp

However, you are technically using the divs correctly, so I'm happy for you to keep them as you have done! :)